interview-questions.md

```diff
@@ -7,44 +7,70 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
+  Your answer: I think we just generate a migration file from the terminal. After that, we locate the migration file. I'm pretty sure the migration is for the student model, which needs to be indicated. I think "rails magic" is what actually generates the foreign key. We also need to indicate that the foreign key connects the student model to the Cohort model. Afterwards we run a rails db:migrate.
+
+  Researched answer: So in our migration, we can add a line to our Student schema like "belongs_to :cohort" and then migrate it. The foreign key will occur on the student model. An experienced friend of mine explained it to me with something as silly as writing your name on your items. To my understanding, the foreign key on the Student model is the indicator for the Cohort model.
```

Comment:
Yes, the actual primary key from the cohort model will become the foreign key for the student model. We would use migration to update the student table 
cohort_id: integer
I like that analogy of thinking of the foreign key like writing your name on an item.